src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -17,17 +17,35 @@
  */
 public class ShooterSubsystem extends SubsystemBase {
 
-    private WPI_TalonFX flywheelMotor;
+    private WPI_TalonFX mainFlywheelMaster;
+    private WPI_TalonFX mainFlywheelSlave;
+    private WPI_TalonFX outerFlywheelMaster;
+    private WPI_TalonFX outerFlywheelSlave;
 
-    private double lastFlywheelSpeed = 0;
-    private double flywheelSpeedROC = 0; // Rate of change of the flywheel speed
+    private double lastMainFlywheelSpeed = 0;
+    private double lastOuterFlywheelSpeed = 0;
+    private double mainFlywheelSpeedROC = 0; // Rate of change of the flywheel speed
+    private double outerFlywheelSpeedROC = 0;
     private Timer t;
 
     public ShooterSubsystem () {
-        flywheelMotor = new WPI_TalonFX(Constants.Shooter.FLYWHEEL_MOTOR_ID);
-        flywheelMotor.configFactoryDefault();
-        flywheelMotor.setNeutralMode(NeutralMode.Coast);
-        PIDParameters.configureMotorPID(flywheelMotor, Constants.Shooter.FLYWHEEL_PID);
+        // Configure main flywheel motors
+        mainFlywheelMaster = new WPI_TalonFX(Constants.Shooter.MAIN_FLYWHEEL_MASTER_ID);
```

Comment:
Avoid copy-pasta here, and this out into a function that you call twice, passing in the ID's and PID
e.g.
```
// Main
ConfigureFlywheel(
       Constants.Shooter.MAIN_FLYWHEEL_MASTER_ID
     , Constants.Shooter.MAIN_FLYWHEEL_SLAVE_ID
     , Constants.Shooter.MAIN_FLYWHEEL_PID);

// Outer
ConfigureFlywheel(
       Constants.Shooter.OUTER_FLYWHEEL_MASTER_ID
     , Constants.Shooter.OUTER_FLYWHEEL_PID
     , Constants.Shooter.OUTER_FLYWHEEL_SLAVE_ID);

```

Reply:
Duly noted, thanks